src/utils/reset-password-form.html

```diff
@@ -112,7 +112,7 @@ <h2>Password Reset</h2>
             newPassword: newPassword,
           };
 
-          fetch('http://localhost:3000/auth/reset-password', {
+          fetch('http://13.49.21.92:3000/auth/reset-password', {
```

Comment:
this shouldn't be hardcoded - use environment variables

Reply:
I mean the server IP / domain address of course (+ port)

Reply:
you might also want to include `http://` too, as on the server you should use `https://`, while locally localhost most probably runs on `http://`

Reply:
As I mentioned in description, this is 'raw' html. And i think i cannot pass env variables to it (this form is being hosted using Apache HTTP Server)